rustls/src/lib.rs

```diff
@@ -63,10 +63,10 @@
 //! support WebAssembly.
 //! For more information, see [the supported `ring` target platforms][ring-target-platforms].
 //!
-//! By providing a custom instance of the [`crate::crypto::CryptoProvider`] struct, you
+//! By providing a custom instance of the crypto::CryptoProvider struct, you
```

Comment:
So I guess the syncing of crate root docs and README is a good reason to keep them there in the README, too?

Reply:
I don't understand the question.